src/ws.nim

```diff
@@ -0,0 +1,415 @@
+import chronos, httputils, strutils, base64, std/sha1, random, http,
```

Comment:
Its unclear what `random` is going to be used from standard library or from `src` folder.

Reply:
Fixed it to use local `./random`